content/SCALE/SCALETutorials/Shares/SMB/AddSMBHomeShare.md

```diff
@@ -14,103 +14,159 @@ They are not recommended for new deployments.
+Resolve any issues with Active Directory before proceeding. If Active Directory cannot bind with TrueNAS you cannot start the SMB service after making changes.
 
-![AddUserDirPermsAuthSettings](/images/SCALE/Credentials/AddUserDirPermsAuthSettings.png "Add User Directories, Permissions and Authentication Settings")
+To add private shares and datasets for users that require home directories:
 
-If existing users require access to the home share, go to **Credentials > Users** and edit an existing account.
+1. Create the share using the **Private SMB Datasets and Shares** preset.
 
-Adjust the user home directory to the appropriate dataset and give it a name to create its own directory.
+2. Configure the share dataset ACL to use the **NFSv4_HOME** preset.
 
-{{< hint type="important" title="Home Directory Known Impacts" >}}
-{{< include file="/static/includes/24.04HomeDirectory.md" >}}
+3. Create users either manually or through Active Directory.
 
-{{< expand "Why the change?" "v" >}}
-TrueNAS uses the `pam_mkhomdir` PAM module in the pam_open_session configuration file to automatically create user home directories if they do not exist.
-`pam_mkhomedir` returns `PAM_PERM_DENIED` if it fails to create a home directory for a user, which eventually turns into a pam_open_session() failure.
-This does not impact other PAM API calls, for example, `pam_authenticate()`.
+### Creating the Share and Dataset
 
-TrueNAS SCALE does not include the customized version of `pam_mkhomedir` used in TrueNAS CORE that specifically avoided trying to create the `/nonexistent` directory. This led to some circumstances where users could create the `/nonexistent` directory on SCALE versions before 24.04.
+{{< include file="/static/includes/LocalSMBUser.md" >}}
 
-Starting in SCALE 24.04 (Dragonfish), the root filesystem of TrueNAS is read-only, which prevents `pam_mkhomdir` from creating the `/nonexistent` directory in cases where it previously did.
-This results in a permissions error if `pam_open_session()` is called by an application for a user account that has **Home Directory** set to **/nonexistent**.
-{{< /expand >}}
-{{< /hint >}}
+You can use an existing dataset for the share or create a new dataset.
+You can either add a share when you [create the dataset]({{< relref "DatasetsSCALE.md" >}}) for the share on the **Add Dataset** screen, or create the dataset when you add the share on the **Add SMB** screen.
+If creating a simple SMB share and dataset use either method, or if customizing the dataset, use the [**Add Dataset** screen]({{< relref "DatasetsSCALE.md" >}}) to access dataset advanced setting options.
+To configure a customized SMB share, use the **Add SMB** share option that provides access to the advanced setting options for shares.
+This procedure covers creating the share and dataset from the **Add Share** screen.
 
+To create an alternative to the legacy SMB home share:
```

Comment:
I think once we've had the intro section, we should stick with the new terminology here. Something like "To create a share with private home datasets"

Reply:
I think you are misunderstanding this preset, the share and the dataset are both private not just the dataset. And if I'm correctly associating this comment with the section above, this is about creating the dataset and the share, or in this case, setting up the share to use an existing dataset and provision them as private.